tests/installation/grub_test.pm

```diff
@@ -59,6 +59,10 @@ sub run {
     }
 
     if (get_var("STORAGE_NG") && get_var("ENCRYPT")) {
+        # bootloader timeout is disable so hit 'ret' is needed
+        assert_screen 'grub2';
+        send_key 'ret';
+
```

Comment:
side-note: We are doing the same as in opensusebasetest::wait_boot. Should use DRY.

Also, that's not very adaptive because here we now rely on disable_grub_timeout being active. If we disable it the following will fail.

Eventually we should use something like https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3833